app/components/works/show/authors_show_component.rb

```diff
@@ -4,12 +4,12 @@ module Works
   module Show
     # Component for rendering a table of authors on the work show page.
     class AuthorsShowComponent < ApplicationComponent
-      def initialize(work_presenter:)
-        @work_presenter = work_presenter
+      def initialize(authors:)
```

Comment:
Small tweak: I think the authors show component cares more about the authors (from the presenter) than the presenter itself, so inject them directly and limit knowledge of the presenter to the works controller & show template.